### PR TITLE
[WIP] production instructions.

### PR DESCRIPTION
**Pull request by julialongtin**
(fill in your PR description)

## Checklist:

Please tick the following before making your PR:

* [x] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [x] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

File: src/how-to/install/helm-prod.rst [hunk 1]
```diff
@@ -8,7 +8,336 @@ Installing wire-server (production) components using helm
...
+  │   └── values.yaml
+  └── my-wire-server
+      ├── secrets.yaml
+      └── values.yaml
+
+Inside the ``nginx-ingress-services`` directory, open ``values.yaml`` and replace ``example.com`` with a domain of your choosing. You can try using ``sed -i 's/example.com/<your-domain>/g' values.yaml``.
+
+Next, open ``secrets.yaml`` and add a TLS wildcard certificate and private key matching your domain. For ``example.com``, you need a certficate for ``*.example.com``. The easiest and cheapest options are:
+
+1. use `Let's Encrypt <https://letsencrypt.org/getting-started/>`__
```
Comment (lucendio) [inline src/how-to/install/helm-prod.rst:210]:
```suggestion
Next, open ``secrets.yaml`` and add a TLS wildcard certificate and private key matching your domain. For ``example.com``, you need a certficate for ``*.example.com``. The easiest and cheapest option is `Let's Encrypt <https://letsencrypt.org/getting-started/>`__
```
this one item in the list looks a bit lonely ;)

File: src/how-to/install/helm-prod.rst [hunk 2]
```diff
@@ -8,7 +8,336 @@ Installing wire-server (production) components using helm
...
+
+Type ``helm version``, you should, if everything is configured correctly, see a result like this:
+
+::
+
+    Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+    Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+
+
+In case ``kubectl version`` shows both Client and Server versions, but ``helm version`` does not show a Server version, you may need to run ``helm init``. The exact version (assuming `v2.X.X` - at the time of writing v3 is not yet supported) matters less as long as both Client and Server versions match (or are very close).
```
[the remark this answers is not in the document]
Comment (tofutim) [reply src/how-to/install/helm-prod.rst:56]: also helm 3 does not have init to add to the confusion

File: src/how-to/install/helm-prod.rst [hunk 3]
```diff
@@ -8,7 +8,336 @@ Installing wire-server (production) components using helm
    Code in this repository should be considered *beta*. As of 2019, we do not (yet)
    run our production infrastructure on kubernetes (but plan to do so soon).
 
+Introduction
+-----------------
 
-TODO: This section will be documented soon...
+The following will install a version of all the wire-server components. This setup is not recommended in production but will get you started.
```
[the remark this answers is not in the document]
Comment (tofutim) [reply src/how-to/install/helm-prod.rst:14]: what makes it not recommended in production?

**Comment by julialongtin** [comment]
@tofutim I have no idea who you are, but i'll take another review. :)

File: src/how-to/install/helm-prod.rst [hunk 4]
```diff
@@ -1,14 +1,395 @@
...
+(You can see available helm charts by running ``helm search wire/``. To see
+new versions as time passes, you may need to run ``helm repo update``)
+
+Great! Now you can start installing.
+
+.. note::
+
+    all commands below can also take an extra ``--namespace <your-namespace>`` if you don't want to install into the default kubernetes namespace.
+
+There is a shell script for doing the following procedure with helm 2, for reference. examine wire-server-deploy/bin/prod-setup.sh .
```
Comment (lucendio) [inline src/how-to/install/helm-prod.rst:107]:
```suggestion
There is a shell script for doing the following procedure with helm 2. For reference, examine `prod-setup.sh <https://github.com/wireapp/wire-server-deploy/blob/develop/bin/prod-setup.sh>`__.
```
Not sure about the syntax, though.

File: src/how-to/install/helm-prod.rst [hunk 5]
```diff
@@ -1,14 +1,395 @@
...
+.. code:: shell
+
+   kubectl get pods -w
+
+This will block your terminal and show some things happening as you proceed through this guide. Keep this terminal open and open a second terminal.
+
+How to install charts that provide access to external databases
+---------------------------------------------------------------
+--------------------------------------------------------------------------------
+Before you can deploy the helm charts that tell wire where external services
```
Comment (lucendio) [inline src/how-to/install/helm-prod.rst:123]:
```suggestion
How to install charts that provide access to external databases
---------------------------------------------------------------

Before you can deploy the helm charts that tell wire where external services
```

File: src/how-to/install/helm-prod.rst [hunk 6]
```diff
@@ -1,14 +1,395 @@
...
+As long as nobody is using your cluster yet, you can safely delete and re-create a specific helm release (list releases with ``helm list --all``). Example delete the ``wire-server`` helm release:
+
+.. code:: shell
+
+    helm delete --purge wire-server
+
+It doesn't work, but my problem isn't listed here. Help!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Feel free to open a github issue or pull request `here <https://github.com/wireapp/wire-docs>`_ and we'll try to improve the documentation.
```
Comment (lucendio) [inline src/how-to/install/helm-prod.rst:395]: I'm undecided whether we want to point people to the docs issues or wire-server-deploy repo. Any thoughts? If people have problems deploying things, for which the code foundation resided in wire-server-deploy they might look for similar issues there and not here, wont they?

**Pull request merged**